lib/vmdb/plugins.rb

```diff
@@ -11,6 +11,19 @@ def initialize
       @vmdb_plugins = []
     end
 
+    def vmdb_plugins
+      @vmdb_plugins.empty? ? register_from_railties : @vmdb_plugins
+    end
+
+    def register_from_railties
```

Comment:
Minor, but I think I'd prefer the method to be called `register_vmdb_plugins`. 

- The name matches the initializer name
- The name mirrors the singular method `register_vmdb_plugin`, so you know it's registering all of them
- that the plugins are found via railties now is an implementation detail and is subject to possibly change.